src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,11 @@
+@mixin onTablet {
+  @media (min-width: 744px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width: 1024px) {
```

Comment:
It's better to use a variable for the desktop breakpoint value (e.g., $desktop-breakpoint) instead of hardcoding it.